cc_utilities/command_line/sync_redcap_to_commcare.py

```diff
@@ -117,8 +118,10 @@ def main_with_args(
         if len(redcap_records.index) == 0:
             logger.info("No records returned from REDCap; aborting sync.")
         else:
+            logger.info(f"Found {len(redcap_records)} REDCap records to sync.")
             complete_records, incomplete_records = (
-                redcap_records.pipe(collapse_checkbox_columns)
+                redcap_records.pipe(populate_symptom_columns)
```

Comment:
A comment here to ensure this order isn't inadvertently changed might be nice, e.g.:

```suggestion
                # populate_symptom_columns must come before collapse_checkbox_columns, since
                # it relies on the data structure of a set of checkboxes ("symptoms_selected")
                # before it's collapsed
                redcap_records.pipe(populate_symptom_columns)
```